open-api/rest-catalog-open-api.yaml

```diff
@@ -846,6 +846,125 @@ paths:
         5XX:
           $ref: '#/components/responses/ServerErrorResponse'
 
+  /v1/{prefix}/transactions/commit:
```

Comment:
I like this route. Maybe we should base the method name on it? `catalog.commitTransaction`?

Reply:
that makes actually sense as well. I'll update this accordingly